README.md

```diff
@@ -62,14 +66,36 @@ There are _some_ sense to the default keybindings:
 Although initially designed for giving lectures, the whiteboard controls are pretty good.
 I might add support for extending infinitely in several directions, in the style of a regular whiteboard app.
 
+## Frequently asked questions
+
+> My saved file won't open. Help!
+
+There was a short period of time when we used a different file format for our JSON files.
+It's very unlikely that you have such a file.
+In case you do, you can make it compatible with the modern qboard app by taking the file and wrapping the contents like so:
+
+```json
+{
+  "qboard-version": 1,
+  "pages": OLD_FILE_CONTENTS_GO_HERE
+}
+```
+
+If we have released a new file version beyond version 1, just opening any old files and saving them again will update them to the latest version.
+
 ## Implementation details
 
 It's build on the [nwb](https://github.com/insin/nwb) toolkit, which handles React, Webpack, and Babel.
 We're using Typescript.
 The main app is mostly powered through [Fabric.js](http://fabricjs.com/), with [KeyboardJS](https://github.com/RobertWHurst/KeyboardJS) handling keybindings, and [pdfmake](http://pdfmake.org/#/) handling exporting to PDF.
 
 We extend the Fabric canvas to a [Page class](src/lib/pages.ts) with some convenience functions.
-The Pages class stores pages in a JSON array; whenever we switch pages, we remove all the objects in the canvas and reload from memory.
+The Pages class stores pages in a JSON array;
+whenever we switch pages, we remove all the objects in the canvas and reload from memory.
+In other words, we only store the live objects for the current page;
+all other pages are stored serialized.
+Boards are serialized to JSON just by collecting the serialized array, and adding a small amount of metadata for compatibility.
```

Comment:
`for compatibility` --> `to ensure compatibility`?

Reply:
yeah